completions/reportbug

```diff
@@ -27,41 +27,19 @@ _reportbug()
                 done)' -- "$cur"))
             return
             ;;
-        --bts | -!(-*)B)
-            COMPREPLY=($(compgen -W "debian guug kde mandrake help" -- \
-                "$cur"))
+        --tag | --ui | --interface | --type | --bts | --severity | --mode | -!(-*)[TutBS])
+            COMPREPLY+=($(
+                compgen -W \
+                    '$("$1" $prev help 2>&1 | sed -ne /^[[:space:]]/p)' \
```

Comment:
```suggestion
                    '$("$1" $prev help 2>&1 </dev/null | sed -ne "/^[[:space:]]/p")' \
```

I also suggest quoting the regex passed to `sed`. I noticed that `sed -ne /^[[:space:]]/p` produces an error message and outputs the help text of `sed` in my Debian 10. This is caused by `nullglob`: the word `/^[[:space:]]/p` contains a glob pattern `[[:space:]]` matching nothing, so `sed` receives just an option `-ne` as its commandline arguments.

Reply:
Good catch, thanks.

Reply:
I'm sorry. This workaround turned out to fail to produce the candidates. I expected that `help` should output candidates even without stdin, but it seems to fail when stdin is redirected `/dev/null`. I'll think about an alternative workaround.